addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js

```diff
@@ -37,7 +37,7 @@ export class ProjectTaskKanbanRenderer extends KanbanRenderer {
     }
 
     isProjectTasksContext() {
-        return this.props.list.context.active_model === "project.project" && this.props.list.context.default_project_id;
```

Comment:
I think the first part of the return is there for a reason. Instead of removing it, i'd suggest modifying as follow :
```js
return this.props.list.context.active_model in ["project.project", "project.task.type.delete.wizard"] && this.props.list.context.default_project_id;
```